app/services/foreman/plugin.rb

```diff
@@ -93,6 +93,17 @@ def installed?(id)
 
     def initialize(id)
       @id = id.to_sym
+      add_loggers
+    end
+
+    def add_loggers
+      return unless (defined?(Foreman::Logging) && SETTINGS[@id])
+
+      if (loggers = SETTINGS[@id][:loggers])
```

Comment:
Seems like we need a mechanism for default configuration which doesn't rely on SETTINGS, or a way to prepopulate SETTINGS with that default configuration, else plugins aren't going to be configured well by default - especially on upgrade if config files aren't updated.

It's also a different configuration mechanism to Foreman core which appears to now use the logging.yaml file, which could be confusing for users.